### PR TITLE
adding House_Data to Branch1

### PR DESCRIPTION
**Pull request by chiarakrishnareddy**

### CODE REVIEW

File: House_Data/lib.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    print(type(zip_code))
+    merge_zip = zip_code.merge(merge, left_on = 'ZCTA5CE10', right_on = 'zip')
+    geo_merge_zip = gpd.GeoDataFrame(merge_zip)
+    geo_merge_zip.set_geometry('geometry_x', inplace = True)
+    geo_merge_zip["price"] = geo_merge_zip['price'].apply(lambda x: x/1000)
+    print(type(geo_merge_zip))
+    norm = Normalize(vmin = geo_merge_zip['price'].min(), vmax = geo_merge_zip['price'].quantile(0.95))
+    geo_merge_zip.plot(ax = ax, edgecolor = 'black', linewidth = 1.0, column = 'price', cmap = 'Reds', legend = True, norm = norm)
+
+    print(geo_merge_zip.columns)
```
Comment (finben) [inline House_Data/lib.py:48]: do you still need this print statement?!

File: House_Data/questions.txt
Comment (finben) [inline House_Data/questions.txt:1]: Do we really need this in the project?

File: House_Data/script.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+#import geopandas as gpd
+from matplotlib.colors import Normalize
+from sklearn.linear_model import LinearRegression
+from sklearn.metrics import mean_squared_error , r2_score
+from sklearn.model_selection import train_test_split
+import lib as lb
+
+
+pd.options.display.float_format = '{:.10f}'.format
+df_housing = pd.read_csv('~/desktop/House_Data/USA_Housing.csv')
```
Comment (finben) [inline House_Data/script.py:15]: This file path will not work except for on your local machine

**Review by finben — COMMENTED** [review]
Mostly looks great! Just a few nit picks!

File: House_Data/questions.txt
Comment (chiarakrishnareddy) [reply House_Data/questions.txt:1]: deleted the file

**Pull request merged**